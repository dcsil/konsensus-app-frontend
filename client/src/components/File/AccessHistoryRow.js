import { Badge } from 'reactstrap';
import Avatar from 'components/Profile/Avatar';
import Moment from 'react-moment';

const AccessHistoryRow = ({ user, action, time }) => {
  return (
    <tr>
      <td>
        <div className="avatar-group">
          <a
            className="avatar avatar-sm"
            href="#pablo"
            id="tooltip742438047"
            onClick={(e) => e.preventDefault()}
          >
            <Avatar name={`${user.firstName} ${user.lastName}`} url={user.image}/>
          </a>
        </div>
      </td>
      <td>
        <span className="mb-0 text-sm">{`${user.firstName} ${user.lastName}`}</span>
      </td>
      <td>{user.role}</td>
      <td>{action}</td>
      <td>
        <Badge color="" className="badge-dot mr-4">
          <i className="bg-success" />
          secure
        </Badge>
      </td>

      <td>
        <Moment format="MMM DD/YYYY hh:mm" date={time} />
      </td>
    </tr>
  );
};

export default AccessHistoryRow;
