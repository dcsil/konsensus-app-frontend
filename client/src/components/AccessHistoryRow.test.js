import React from 'react';
import { shallow } from 'enzyme';
import AccessHistoryRow from './File/AccessHistoryRow';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });

test('Test AccessHistoryRow component', () => {
  const wrapper = shallow(<AccessHistoryRow />);
  expect(wrapper).toMatchSnapshot();
});
