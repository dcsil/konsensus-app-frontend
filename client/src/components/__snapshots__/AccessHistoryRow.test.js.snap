// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test AccessHistoryRow component 1`] = `ShallowWrapper {}`;
